sunpy/visualization/imageanimator.py

```diff
@@ -878,3 +875,117 @@ def update_plot(self, val, line, slider):
         if val != slider.cval:
             line.set_ydata(self.data[self.frame_slice])
             slider.cval = val
+
+class ImageAnimatorWCS(ImageAnimator):
+    """
+    Create a matplotlib backend independent data explorer for 2D images that uses wcs data.
```

Comment:
something like "for arrays with associated astropy WCS objects" is more descriptive.